pkgs/applications/editors/emacs-modes/melpa-packages.nix

```diff
@@ -235,6 +235,12 @@ self:
       # upstream issue: missing file header
       textmate = markBroken super.textmate;
 
+      treemacs-magit = super.treemacs-magit.overrideAttrs (attrs: {
+        # searches for Git at build time
+        nativeBuildInputs =
+          (attrs.nativeBuildInputs or []) ++ [ external.git ];
```

Comment:
The `()` aren't necessary here

Reply:
This was just copied from the other overrides that [use the extra parenthesis](https://github.com/NixOS/nixpkgs/blob/6ec937ee839b06ffaa122acce6f48dbc44e47721/pkgs/applications/editors/emacs-modes/melpa-packages.nix#L135). In the interest of consistency, should I keep the parens in this case or remove them from the other overrides as well?

Reply:
Either is fine by me

Reply:
I’ll keep it as is then, to keep the diff smaller.